frontend/src/components/CreateListing/CreateListingButton.jsx

```diff
@@ -153,6 +157,7 @@ class CreateListingButton extends Component {
                 let url = API_ROOT_POST.concat("listing")
                 let body = {
                   images: imageCollection,
+                  userid: this.props.cookies.get("userid"),
```

Comment:
can we change this :p you don't need to change other places, but userid will be used for other routes like `GET user/:id/listing`. This only applies to POST route.

Reply:
probably deal with this in the future. we will assume people will not intentionally call `GET user/:id/listing` with inconsistent userid for now (although it can be bad :|)